controllers/articles.controller.js

```diff
@@ -0,0 +1,19 @@
+const { fetchArticle } = require("../models/index")
+
+exports.getArticle = (request, response, next) => {
+    const { article_id } = request.params
+    if (isNaN(article_id)){
```

Comment:
perhaps unnecessary as Postgres will throw an error if the passed id is not a found in the query anyway